src/libgraphviz/lib.rs

```diff
@@ -531,29 +533,36 @@ pub enum RenderOption {
 }
 
 /// Returns vec holding all the default render options.
-pub fn default_options() -> Vec<RenderOption> { vec![] }
+pub fn default_options() -> Vec<RenderOption> {
+    vec![]
+}
```

Comment:
I do like having the whole fn sig and body in one line when the body really is short.  Is this hard to accommodate, @nrc ?  Or is this more of a philosophical thing about "bodies should be presented separately from signatures" ?